src/components/ProductsTable/ProductsTable.tsx

```diff
@@ -251,7 +254,7 @@ const ProductsTable: FunctionComponent<ProductsTableProps> = ({
           ))}
         </Tbody>
       </TableComposable>
-      {paginatedProducts.length == 0 && data.length > 0 && (
+      {paginatedProducts.length == 0 && data.length >= 0 && (
```

Comment:
I'm confused about the logic of this if check. Shouldn't it show no results if either of these values are strictly equal to 0? Also is there ever a case where paginatedProducts.length is not 0 when data.length is 0? I feel like it works if we just check len of paginated products

Reply:
I've removed all logic minus the paginatedProducts.length logic, and it seems to work the same. I can push with just that logic

Reply:
Cool, yeah if the behavior is the same I think that's a cleaner and more correct implementation